pkg/printer/enclave.go

```diff
@@ -0,0 +1,363 @@
+/*
```

Comment:
The commit `chore: move enclave print functions to separate file` is purely moving functions- there aren't any logic changes.